pkg/api/config.go

```diff
@@ -210,6 +216,19 @@ func validateTestConfigurationType(fieldRoot string, test TestStepConfiguration)
 		if len(test.From) > 0 {
 			validationErrors = append(validationErrors, fmt.Errorf("%s specifies both `From` and a test type", fieldRoot))
 		}
+		if needsReleaseRpms {
+			var version []int
+			if release != nil {
+				if match := originReleaseTagRegexp.FindStringSubmatch(release.Name); match != nil {
+					version = make([]int, 2)
+					version[0], _ = strconv.Atoi(match[1])
+					version[1], _ = strconv.Atoi(match[2])
+				}
+			}
+			if version == nil {
```

Comment:
since you are checking if there is no version, why you populate the []int above?
Can you append the error in `if match := originReleaseTagRegexp.FindStringSubmatch(release.Name); match == nil {` ?